### ISSUE TITLE
[Feature]:  Aspect Ratio

### ISSUE THREAD

**Issue by fairysubsteam** [movie-web/movie-web#425]
### What feature do you want to add?

Aspect Ratio button

### Why do you want to have this feature?

Change the aspect ratio of the video to the most used screen formats like letterbox 4:3, 16:9, 18:9, 21:9, and some others like fit height, fit width, fill screen... Etc

**Comment by castdrian** [comment]
I think I vaguely remember an internal discussion for player zoom in conjuction with auto-rotate and fullscreen stuff, probably relevant to, probably covered by #338

**Comment by mrjvs** [comment]
its not covered by #338, so ill keep this issue open. I can see users may be wanting this in the future.

**Comment by castdrian** [comment]
yeah I think I have specifically mentioned it myself since I do have a very odd aspect ratio to deal with, I can see at minimum fit to screen and zoom to fill being things that people would greatly appreciate

**Comment by dhruvit07** [comment]
Hi there,

I'm a developer who has been eagerly waiting for a feature like this, especially for Samsung TV compatibility. None of the existing streaming apps seem to cater to this need, so I'd love to take the initiative and contribute to implementing the Aspect Ratio button.

Could you please grant me permission to work on this feature? Additionally, I'd appreciate some guidance on how I can start contributing to this project. Looking forward to collaborating on this!

Best,
Dhruvit

**Comment by fairysubsteam** [comment]
Most of the features are awaiting approval, and I don't think any of them will be added any time soon, nor would the devs allow any contributions.

**Comment by dhruvit07** [comment]
Why not allow the contribution? Any reason?

**Comment by zisra** [comment]
@dhruvit07 the approved label has been added, so you can surely work on this

**Comment by mrjvs** [comment]
I've assigned, please add the setting under "playback settings", below the video speed

**Comment by dhruvit07** [comment]
Got it! Thanks!

**Comment by zisra** [comment]
@dhruvit07 any update?

**Comment by binaryoverload** [comment]
Unassigning as no progress - this is free to be picked up by someone else
